lib/puppet/agent/locker.rb

```diff
@@ -29,11 +29,11 @@ def lock
 
   # @deprecated
   def running?
-    Puppet.deprecation_warning _(<<-ENDHEREDOC)
-Puppet::Agent::Locker.running? is deprecated as it is inherently unsafe.
-The only safe way to know if the lock is locked is to try lock and perform some
-action and then handle the LockError that may result.
-ENDHEREDOC
+    #TRANSLATORS 'Puppet::Agent::Locker.running?' is a method name and should not be translated
+    message = _('Puppet::Agent::Locker.running? is deprecated as it is inherently unsafe.')
+    message += ' ' + _('The only safe way to know if the lock is locked is to try lock and perform some'\
+                       'action and then handle the LockError that may result.')
+    Puppet.deprecation_warning _(message)
```

Comment:
do you still need `_()` to wrap message here? I was under the impression the previous lines negated the need for this?

Reply:
Fixed